docs/content/examples/tutorials/crowdsec.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Tutorials | Crowdsec'
+---
+
+!!! quote "What is Crowdsec ?"
+    
+    Crowdsec is an open source software that detects and blocks attackers using log analysis.
+    It has access to a global community-wide IP reputation database.
+
+    [Source][crowdsec]
+
+    [crowdsec]: https://www.crowdsec.net
+
+## Installation
+
+Crowdsec supports multiple [installation methods][crowdsec-installation-docs], however this page will use the docker installation.
+
+
+### Docker mailserver
+
+In the docker-compose definition of the docker-mailserver, create a bind volume containing the logs.
+
+```yaml
+services:
+  mailserver:
+    image: ghcr.io/docker-mailserver/docker-mailserver:latest
+    container_name: mailserver
+    # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
+    hostname: mail.example.com
+    ports:
+      - "25:25"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+    volumes:
+      - /docker-data/dms/mail-data/:/var/mail/
+      - /docker-data/dms/mail-state/:/var/mail-state/
+      - /docker-data/dms/mail-logs/:/var/log/mail/
+      - /docker-data/dms/config/:/tmp/docker-mailserver/
+      - /etc/localtime:/etc/localtime:ro
+    environment:
+      - ENABLE_RSPAMD=1
+      - ENABLE_CLAMAV=1
+      - ENABLE_FAIL2BAN=0
+    restart: always
+```
+
+### Crowdsec
+
+The crowdsec container needs to have access to the docker-mailserver logs.
```

Comment:
I'm fine with the yaml `compose.yaml` snippet here as this may be less familiar to our docs readers.

```suggestion
The crowdsec container should also bind mount the same host path for DMS logs that was added in the DMS example above.
```